ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentDefinitionPreparer.scala

```diff
@@ -55,11 +58,11 @@ object ComponentDefinitionPreparer {
 
     //TODO: make it possible to configure other defaults here.
     val base = ComponentGroup(BaseGroupName, List(
-      ComponentTemplate.create(ComponentType.Filter, Filter("", Expression("spel", "true")), readCategories),
-      ComponentTemplate.create(ComponentType.Split, Split(""), readCategories),
-      ComponentTemplate.create(ComponentType.Switch, Switch("", Expression("spel", "true"), "output"), readCategories),
-      ComponentTemplate.create(ComponentType.Variable, Variable("", "varName", Expression("spel", "'value'")), readCategories),
-      ComponentTemplate.create(ComponentType.MapVariable, VariableBuilder("", "mapVarName", List(Field("varName", Expression("spel", "'value'")))), readCategories),
+      ComponentTemplate.create(ComponentType.Filter, Filter("", Expression("spel", "true")), userCategories),
```

Comment:
should it be `userProcessingTypeCategories`?

Reply:
Done